### ISSUE TITLE
Parameter types incorrect when using default values

### ISSUE THREAD

**Issue by epotter2297** [ecyrbe/zodios-express#64]
If I have an API definition with a parameter like this:
```js
{
  name: 'limit',
  type: 'Query',
  schema: z.number().positive().default(10),
},
```
In my router, `req.query.limit` is of type `number | undefined`. I would expect it to be `number` due to the default value.

**Comment by ecyrbe** [comment]
Oh yes, this one is tricky. Thanks for the report.
  
In frontend, the type is correct, because it allows effectively you to not pass any value.
  
But in the backend, you effectively expect the value to be generated, so yes indeed, you expect it to be `number`.

I'll need to extend `@zodios/core` to add support for this.

**Comment by ecyrbe** [comment]
@epotter2297 This is now fixed in version `@zodios/core` `v10.4.4` and `@zodios/express` `v10.4.3`
